support-frontend/assets/helpers/__tests__/contributionsTests.ts

```diff
@@ -13,52 +14,63 @@ import {
 	NZDCountries,
 	UnitedStates,
 } from '../internationalisation/countryGroup';
+
 // ----- Tests ----- //
+
 describe('contributions', () => {
 	describe('validateContribution', () => {
 		it('should return "TooLittle" for contributions below the minimum', () => {
-			[
-				[1, 'MONTHLY', GBPCountries],
-				[5, 'ANNUAL', AUDCountries],
-				[0.5, 'ONE_OFF', EURCountries],
-			].forEach((a) => expect(validateContribution(...a)).toBe('TooLittle'));
+			expect(validateContribution(1, 'MONTHLY', GBPCountries)).toBe(
+				'TooLittle',
+			);
+			expect(validateContribution(5, 'ANNUAL', AUDCountries)).toBe('TooLittle');
+			expect(validateContribution(0.5, 'ONE_OFF', EURCountries)).toBe(
+				'TooLittle',
+			);
```

Comment:
I was struggling to get typescript to be happy with the expansion, so just unrolled the loop